### PR TITLE
Find the total size of largest tables

### PR DESCRIPTION
**Pull request by phumpal**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [ ] Update Changelog following the conventions laid out  [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

**Comment by phumpal** [comment]
@majormoses gentle ping (no rush).

Curious if I shouldn't use `pg_relation_size` instead of `pg_total_relation_size` 

https://www.postgresql.org/docs/current/functions-admin.html#FUNCTIONS-ADMIN-DBOBJECT

**Review by majormoses — APPROVED** [review]
LGTM

File: bin/metric-postgres-relation-size.rb [hunk 1]
```diff
@@ -0,0 +1,119 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: false
+#
```
Comment (majormoses) [inline bin/metric-postgres-relation-size.rb]: Looks like the lint gods demand this to be an empty line vs a comment 🤷 
```suggestion

File: bin/metric-postgres-relation-size.rb [hunk 2]
```diff
@@ -0,0 +1,118 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: false
+#   metrics-postgres-relation-size.rb
+#
```
Comment (majormoses) [inline bin/metric-postgres-relation-size.rb]:
```suggestion

#  metrics-postgres-relation-size.rb
```

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-postgres/versions/4.2.0

**Comment by majormoses** [comment]
> @majormoses gentle ping (no rush).
> 
> Curious if I shouldn't use `pg_relation_size` instead of `pg_total_relation_size`
> 
> https://www.postgresql.org/docs/current/functions-admin.html#FUNCTIONS-ADMIN-DBOBJECT

I am not familiar enough with postgres internals to have an intelligent opinion without spending some time diving, we can always change it later if we think that makes sense.

EDIT: perhaps we can allow an arg to switch the behavior so admins can tell us what they want even if we take _some_ default stance.

**Comment by phumpal** [comment]
@majormoses on 

- pg_total_relation_size - returns the disk space as an aggregate size - table, indexes, compressed data
- pg_relation_size - disk space used by table (excludes indexes)

> perhaps we can allow an arg to switch the behavior so admins can tell us what they want even if we take some default 

Good idea, I will circle back on this in a week or so.

**Comment by majormoses** [comment]
@phumpal I read the descriptions and I lean towards `pg_total_relation_size` which is what you had (hence my merge), not sure if someone more familiar with DB stuff has a good reason why you should not care about index size. If we add the arg to control that then they could run two instances of the same check for each one.

**Pull request merged**